spec/stepmod/converters/eqn_spec.rb

```diff
@@ -56,7 +56,7 @@
       XML
     end
     let(:output) do
-      "directrix::   a circle in 3D space of radius R ~1~ , "
+      "\n\ndirectrix::   a circle in 3D space of radius R _{1} , \n"
```

Comment:
Can we use heredoc/multi-line strings for comparison? It's hard to read. Also, `R_{1}` (need to strip the space right before `_{1}`.

Reply:
Done